src/frameworks/cucumber.js

```diff
@@ -67,11 +72,15 @@ export default {
    */
   onSuiteEnd (suite) {
     if (config.usingAllure) {
-      if (browser.capabilities.deviceType) {
-        allureReporter.addArgument('deviceType', browser.capabilities.deviceType);
+      const capabilities = this.isMultiremote
+      ? this.capabilities[Object.keys(this.capabilities)[0]]
+      : this.capabilities;
```

Comment:
```suggestion
        ? this.capabilities[Object.keys(this.capabilities)[0]]
        : this.capabilities;
```

Reply:
Maybe we can put this assignment into a function, given we do the same above as again

Reply:
Sure, have created a function in helpers.js and used it across. Please review it again and approve running workflow @christian-bromann 
Thanks :)